lib/chat_web/live/main_live/index.ex

```diff
@@ -318,6 +318,13 @@ defmodule ChatWeb.MainLive.Index do
 
   def handle_info({:dialog, msg}, socket), do: socket |> Page.DialogRouter.info(msg) |> noreply()
 
+  def handle_info({ref, {:ok, message}}, socket) do
```

Comment:
And if task fails it bring down client's LiveView?

Reply:
now it catches a fail inside the task

Reply:
applied Task.Supervisor